base_tier_validation/models/tier_review.py

```diff
@@ -150,15 +150,24 @@ def _compute_todo_by(self):
     def _get_reviewers(self):
         if self.reviewer_id or self.reviewer_group_id.users:
             return self.reviewer_id + self.reviewer_group_id.users
-        reviewer_field = self.env["res.users"]
         if self.reviewer_field_id:
             resource = self.env[self.model].browse(self.res_id)
             reviewer_field = getattr(resource, self.reviewer_field_id.name, False)
-            if not reviewer_field or not reviewer_field._name == "res.users":
-                raise ValidationError(
-                    self.env._("There are no res.users in the selected field")
+            if reviewer_field:
+                if reviewer_field._name == "res.groups":
+                    return self._check_reviewers(reviewer_field.users)
+                if reviewer_field._name == "res.users":
+                    return self._check_reviewers(reviewer_field)
+        return self.env["res.users"]
+
+    def _check_reviewers(self, resource):
+        if not resource or resource._name not in ["res.users", "res.groups"]:
```

Comment:
You don't actually seem to pass groups here. Shouldn't resource always be `res.users`?

Reply:
@kevinkhao Just a reminder that this review is still pending to attend